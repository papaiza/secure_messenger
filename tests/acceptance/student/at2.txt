add_user(1, "Joe")
add_user(2, "Dan")
add_group(1, "Friends")
register_user(1, 1)
register_user(2, 1)
send_message(1, 3, "No group with this id")
send_message(3, 1, "No user with this id")
send_message(3, 3, "Neither group nor user id")