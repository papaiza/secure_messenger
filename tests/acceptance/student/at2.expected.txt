  0:  OK

->add_user(1,"Joe")
  1:  OK
  Users:
      1->Joe
  Groups:
  Registrations:
  All messages:
  New messages:
  Old/read messages:

->add_user(2,"Dan")
  2:  OK
  Users:
      1->Joe
      2->Dan
  Groups:
  Registrations:
  All messages:
  New messages:
  Old/read messages:

->add_group(1,"Friends")
  3:  OK
  Users:
      1->Joe
      2->Dan
  Groups:
      1->Friends
  Registrations:
  All messages:
  New messages:
  Old/read messages:

->register_user(1, 1)
  4:  OK
  Users:
      1->Joe
      2->Dan
  Groups:
      1->Friends
  Registrations:
      [1, Joe]->{1->Friends}
  All messages:
  New messages:
  Old/read messages:

->register_user(2, 1)
  5:  OK
  Users:
      1->Joe
      2->Dan
  Groups:
      1->Friends
  Registrations:
      [1, Joe]->{1->Friends}
      [2, Dan]->{1->Friends}
  All messages:
  New messages:
  Old/read messages:

->send_message(0, 3, "ID must be positive")
  6:  ERROR 
  ID must be a positive integer.

->send_message(3, 0, "ID must be positive")
  7:  ERROR 
  ID must be a positive integer.

->send_message(0, 3, "")
  8:  ERROR 
  ID must be a positive integer.

